### PR TITLE
audacity: 3.4.1 -> 3.4.2

### PR DESCRIPTION
**Pull request by wegank**
## Description of changes

https://github.com/audacity/audacity/releases/tag/Audacity-3.4.2

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by wegank** [comment]
I've pushed the changes that work on Darwin; haven't tested on Linux yet.

File: pkgs/applications/audio/audacity/default.nix [hunk 1]
```diff
@@ -30,7 +30,7 @@
 , libid3tag
 , libopus
 , libuuid
-, ffmpeg_4
+, ffmpeg
```
[the remark this answers is not in the document]
Comment (wegank) [reply pkgs/applications/audio/audacity/default.nix]: Done.

**Review by 6t8k — APPROVED** [review]

**Review by 6t8k — APPROVED** [review]

**Comment by wegank** [comment]
I guess I still won't push that commit if https://github.com/NixOS/nixpkgs/pull/267963/commits/fa902960470b40a5efb1862f8e5f0d39709cd863 breaks on Sway without the GTK wrapper.

**Comment by wegank** [comment]
Thanks! I think delaying `postInstall` to `postFixup` would also do the trick, would you please test again?

**Review by 6t8k — APPROVED** [review]

**Pull request merged**